clients/ts/FunctionalTests/rollup.config.js

```diff
@@ -8,7 +8,7 @@ import commonjs from 'rollup-plugin-commonjs'
 import resolve from 'rollup-plugin-node-resolve'
 
 export default {
-    input: path.join(__dirname, "obj", "js", "FunctionalTests", "ts", "index.js"),
+    input: path.join(__dirname, "obj", "js", "index.js"),
```

Comment:
Now that we aren't compiling any of `@aspnet/signalr` during the build of the functional tests, TypeScript doesn't generate the extra folders.